src/index.css

```diff
@@ -70,3 +70,24 @@ select {
   border: none;
   text-align: left;
 }
+
+#admin-page {
+  text-align: center;
+  padding-top: 50px;
+}
+
+.admin-login-title {
+  padding-bottom: 15px;
+}
+
+.login-text-field {
+  --brown-glow-drop-shadow: none;
+
+}
+
+.login-button {
+  padding-top: 25px;
+  --brown-glow-drop-shadow: none;
+
+
+}
```

Comment:
Two things:
* Can we cleanup the empty lines?
* Instead of having to apply this to all the custom classes, you could just make it so nothing on this admin page has the dropshadow styling. Something like:

```
.myClass h1,
.myClass h3,
.myClass p,
.myClass span,
.myClass button,
.myClass select,
.myClass input {
    --brown-glow-drop-shadow: none;
}
```

Reply:
Where .myClass is a class on the top level div for the `login` component

Reply:
Or even slightly cleaner:
```
.myClass * {
    --brown-glow-drop-shadow: none;
}
```